addons/l10n_tr_nilvera_edespatch/models/stock_picking.py

```diff
@@ -0,0 +1,239 @@
+            lambda p: p.country_code == 'TR'
+        ).l10n_tr_nilvera_despatch_type = 'SEVK'
+
+    @api.depends('state')
+    def _compute_l10n_tr_nilvera_despatch_state(self):
+        self.filtered(
+            lambda p: p.state == 'done' and p.country_code == 'TR'
+        ).l10n_tr_nilvera_despatch_state = 'to_send'
+
+    def _l10n_tr_validate_edelivery_fields(self):
+        self.ensure_one()
+        if self.state != 'done':
+            self.l10n_tr_nilvera_edespatch_warnings = {
+                'invalid_transfer_state': {
+                    'message': _("Please validate the transfer first to generate the XML"),
+                }
+            }
+            return
+        if self.l10n_tr_nilvera_despatch_scenario == 'HALTIPI':
+            self.l10n_tr_nilvera_edespatch_warnings = {
+                'invalid_delivery_scenario': {
+                    'message': _("Despatch Scenario as Hal Tipi is currently unsupported in the module."),
+                }
+            }
+            return
+        partners = (
+            self.company_id.partner_id
+            | self.partner_id
+            | self.l10n_tr_nilvera_carrier_id
+            | self.l10n_tr_nilvera_buyer_id
+            | self.l10n_tr_nilvera_seller_supplier_id
+            | self.l10n_tr_nilvera_buyer_originator_id
+        )
+        error_messages = {
+            **partners._l10n_tr_nilvera_validate_partner_details(self.partner_id)
+        }
+        if self.l10n_tr_nilvera_despatch_type == 'MATBUDAN':
+            if not self.l10n_tr_nilvera_delivery_date:
+                error_messages['invalid_matbu_date'] = {
+                    'message': _("Printed Delivery Note Date is required."),
+                }
+            if not self.l10n_tr_nilvera_delivery_printed_number:
+                error_messages['invalid_matbu_number'] = {
+                    'message': _("Printed Delivery Note Number is required."),
+                }
+            elif len(self.l10n_tr_nilvera_delivery_printed_number or "") != 16:
+                error_messages['invalid_matbu_number'] = {
+                    'message': _("Printed Delivery Number must be 16 characters."),
+                }
+
+        invalid_country_drivers = invalid_tckn_drivers = self.env['res.partner']
+        for driver in self.l10n_tr_nilvera_driver_ids:
+            if not driver.country_id or driver.country_id.code != 'TR':
+                invalid_country_drivers |= driver
+            elif not driver.vat or (driver.vat and len(driver.vat) != 11):
+                invalid_tckn_drivers |= driver
+        if drivers := len(invalid_country_drivers):
+            error_messages['invalid_driver_country'] = {
+                'message': _(
+                    "Only Drivers from Türkiye are valid. Please update the Country and enter a valid TCKN in the Tax ID."
+                ),
+                'action_text': _(
+                    _("View %s", drivers == 1 and invalid_country_drivers.name or "Drivers"),
```

Comment:
```suggestion
                    "View %s",
                    drivers == 1 and invalid_country_drivers.name or _("Drivers")
```
Link to pylint failure link to [build](https://runbot.odoo.com/runbot/build/78037722) . You are using _(_("View %s")) which is bad practice